src/utils/AnalyzeFLASHIda.cpp

```diff
@@ -0,0 +1,454 @@
+
+#include <OpenMS/APPLICATIONS/TOPPBase.h>
+#include <OpenMS/FORMAT/MzMLFile.h>
+#include <OpenMS/KERNEL/MSExperiment.h>
+#include <OpenMS/SYSTEM/File.h>
+#include <OpenMS/FORMAT/FileHandler.h>
+
+#include <QFile>
+#include <iomanip>
+
+#include <OpenMS/METADATA/SpectrumLookup.h>
+using namespace OpenMS;
+using namespace std;
+
+//-------------------------------------------------------------
+// Doxygen docu
+//-------------------------------------------------------------
+
+/**
+    @page
+*/
+
+/// @cond
+
+class TOPPAnalyzeFLASHIda :
+    public TOPPBase
+{
+public:
+  TOPPAnalyzeFLASHIda() :
+      TOPPBase("AnalyzeFLASHIda", ".", false)
+  {
+  }
+
+protected:
+  void registerOptionsAndFlags_() override
+  {
+    //registerOutputFileList_("out_spec", "<file for MS1, file for MS2, ...>", {""},
+    //                            "output files (tsv) - spectrum level deconvoluted masses per ms level", false);
+    //
+
+    registerInputFileList_("in", "<files >", {}, "Input files");
+    registerOutputFile_("out", "<file>", "", "");
+    //setValidFormats_("i1n", ListUtils::create<String>("tsv"));
+    //setValidFormats_("i1n", ListUtils::create<String>("tsv"));
+  }
+
+  struct TopPicItem
+  {
+  public:
+    TopPicItem() = default;
+
+    TopPicItem(String in)
+    {
+      str_ = in;
+      vector<String> results;
+      stringstream tmp_stream(in);
+      String str;
+
+      while (getline(tmp_stream, str, ','))
+      {
+        results.push_back(str);
+      }
+      //cout<<str_<<endl;
+      protein_acc_ = results[0];
+      proteform_id_ = stoi(results[1]);
+      rt_ = stod(results[2]);
+      precursor_mass_ = stod(results[4]);
+      precursor_mz_ = stod(results[5]);
+      precursor_intensity = stod(results[6]);
+        mass_intensity_ = stod(results[7]);
+      charge_ = stoi(results[8]);
+      unexp_mod_ = stod(results[9]);
+      q_value_ = stod(results[15]);
+      e_value_ = stod(results[16]);
+    }
+
+    //const String header="Findex,";// = "Findex,ACC,ProID,RT,PrecursorMonoMass,PrecursorAvgMass,PrecursorMz,PrecursorIntensity,PrecursorCharge,PTM,ChargeCos,ChargeSNR,Cos,SNR,ChargeScore,Qscore,Evalue,Class";
+
+    String str_;
+    double rt_;
+    int charge_;
+    double precursor_intensity;
+    double mass_intensity_;
+    double precursor_mass_;
+    double precursor_mz_;
+    int proteform_id_;
+    String protein_acc_;
+    double unexp_mod_;
+    double e_value_;
+    double q_value_;
+
+  };
+
+  ExitCodes main_(int, const char **) override
+  {
+    auto infiles = getStringList_("in");
+    auto outfile = getStringOption_("out");
+    double tol = 20.0;
+    fstream outstream;
+    outstream.open(outfile, fstream::out); //
+
+    MSExperiment map;
+    MzMLFile mzml;
+
+    mzml.setLogType(log_type_);
+    mzml.load(infiles[0], map);
+    auto iso_mzs = DoubleList{126.127725, 127.124760, 128.134433, 129.131468, 130.141141, 131.138176};
```

Comment:
can you document those constants?

Reply:
Timo - I will remove these files - please ignore those in src/utils Sorry for completely forgetting them.